packages/server-core/src/networking/gameserver-provision/gameserver-provision-helper.ts

```diff
@@ -0,0 +1,37 @@
+import { Application } from '../../../declarations'
+import { getFreeGameserver } from '../instance-provision/instance-provision.class'
+
+export const patchGameserverLocation = async (app: Application, locationId) => {
+  if (app.k8DefaultClient) {
```

Comment:
Do we need to check if k8DefaultClient exists? It seemed to work locally when I commented out this if statement; while there's not much use for it locally, I don't think it needs to explicitly not work there.

Reply:
Agreed. Updated